R/neopolars-package.R

```diff
@@ -4,5 +4,6 @@
 ## usethis namespace: start
 #' @import rlang
 #' @importFrom utils findMatches head
+#' @importFrom stats median
```

Comment:
Is this needed?
How about `#' @exportS3Method stats::median`?

Reply:
Didn't know this one, fixed